MIGRATION.md

```diff
@@ -0,0 +1,7 @@
+# Migration
```

Comment:
Hmm, sounds like a possible decision to get rid of all documents in Russian. I think that for such projects as Crafter or Blueprinter, Russian versions of readme and changelog can have essential values due to complexity of projects. So deleting them is something I would not do.

When it comes to the migration guide, may be we can add the Russian version of this file just to be consistent and to avoid questions "why all docs have translation except that" in future

Reply:
@igoradamenko maybe I could merge this as is, and you could add needed doc updates in a separate PR when it is convenient for you. What do you think?

Reply:
Merged, thank you :relaxed: